lib/api/src/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -5116,6 +5088,22 @@ fromApiRedeemer = \case
     ApiRedeemerRewarding (ApiBytesT bytes) (StakeAddress x y) ->
         RedeemerRewarding bytes (RewardAcnt x y)
 
+sealWriteTx :: forall era. Write.IsRecentEra era => Write.Tx era -> W.SealedTx
+sealWriteTx = W.sealedTxFromCardano
+    . Cardano.InAnyCardanoEra (Write.cardanoEra @era)
+    . Write.toCardanoApiTx
+
+toApiSerialisedTransaction
+    :: Write.IsRecentEra era
+    => Maybe ApiSealedTxEncoding
+    -> Write.Tx era
+    -> ApiSerialisedTransaction
+toApiSerialisedTransaction maybeEncoding tx =
+    let
+        encoding = fromMaybe Base64Encoded maybeEncoding
+    in
+        ApiSerialisedTransaction (ApiT $ sealWriteTx tx) encoding
```

Comment:
Just curious, why not:
```suggestion
toApiSerialisedTransaction maybeEncoding tx =
    ApiSerialisedTransaction (ApiT $ sealWriteTx tx) encoding
  where
    encoding = fromMaybe Base64Encoded maybeEncoding
```
(Saves both horizontal and vertical space.)

Reply:
In this specific example I think I like how you see the flow of 
1. `maybeEncoding` introduced (parameter)
2. `maybeEncoding` transformed into `encoding`
3. `encoding` is used

in order, as opposed to 1-3-2 with `where`.